src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -45,6 +45,29 @@ namespace dpdk {
     return std::bitset<CHAR_BIT * sizeof(n)>{n}.count();
   }
 
+  uint16_t eal::get_first_slave_coreid(const char *coremask)
+  {
+    uint64_t ulCoreMask;
```

Comment:
might want to avoid using camelCase and Hungarian naming convention, instead could you use something like `core_mask`.

Reply:
thx, i have modify code style as you said, pls check again.